src/xGadgetron/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ endif ()
 find_package(ISMRMRD)
 if (ISMRMRD_FOUND)
   set(CMAKE_MODULE_PATH "${ISMRMRD_DIR}")
+  set(cGadgetron_LIBRARY_DIRS "${ISMRMRD_LIBRARY_DIRS}" PARENT_SCOPE)
```

Comment:
add a comment why we need it (problem in ISMRMRD config cmake files for version...: library location is not part of the target)